packages/kbn-apm-synthtrace/src/cli/utils/bootstrap.ts

```diff
@@ -40,7 +40,13 @@ export async function bootstrap(runOptions: RunOptions) {
     concurrency: runOptions.concurrency,
   });
 
-  await kibanaClient.installApmPackage(latestPackageVersion);
+  if (!runOptions['skip-package-install']) {
```

Comment:
Shouldn't we skip the entire process if skip-package-install is true?

Reply:
We are skipping the whole installation process. We still need to call the Fleet API to get the package version as its required to populate `observer.version` field

Reply:
solved offline 👍🏼